.nuget/create_nuget.py

```diff
@@ -7,18 +7,22 @@
 if __name__ == "__main__":
     source = sys.argv[1]
     current_dir = os.path.abspath(os.path.dirname(__file__))
-    linux_folder_path = os.path.join(current_dir, "runtimes", "linux-x64", "native")
+    linux_folder_path = os.path.join(current_dir,"lib", "netstandard2.0", "runtimes", "linux-x64", "native")
     if not os.path.exists(linux_folder_path):
         os.makedirs(linux_folder_path)
-    osx_folder_path = os.path.join(current_dir, "runtimes", "osx-x64", "native")
+    osx_folder_path = os.path.join(current_dir, "lib", "netstandard2.0", "runtimes", "osx-x64", "native")
     if not os.path.exists(osx_folder_path):
         os.makedirs(osx_folder_path)
-    windows_folder_path = os.path.join(current_dir, "runtimes", "win-x64", "native")
+    windows_folder_path = os.path.join(current_dir, "lib", "netstandard2.0", "runtimes", "win-x64", "native")
     if not os.path.exists(windows_folder_path):
         os.makedirs(windows_folder_path)
+    net46_folder_path = os.path.join(current_dir, "lib", "net46")
+    if not os.path.exists(net46_folder_path):
+        os.makedirs(net46_folder_path)
     copy_file(os.path.join(source, "lib_lightgbm.so"), os.path.join(linux_folder_path, "lib_lightgbm.so"))
     copy_file(os.path.join(source, "lib_lightgbm.dylib"), os.path.join(osx_folder_path, "lib_lightgbm.dylib"))
     copy_file(os.path.join(source, "lib_lightgbm.dll"), os.path.join(windows_folder_path, "lib_lightgbm.dll"))
+    copy_file(os.path.join(source, "lib_lightgbm.dll"), os.path.join(net46_folder_path, "lib_lightgbm.dll"))
```

Comment:
What about the exe?

Reply:
@codemzs the exe is removed in the latest nuget package, do we still need it ?